beets/autotag/mb.py

```diff
@@ -276,11 +276,18 @@ def album_info(release):
         disambig.append(release.get('disambiguation'))
     info.albumdisambig = u', '.join(disambig)
 
-    # Release type not always populated.
-    if 'type' in release['release-group']:
-        reltype = release['release-group']['type']
-        if reltype:
-            info.albumtype = reltype.lower()
+    # Considers all release types (both primary and secondary) and stores as a comma-separated string
+    all_types = []
+    if 'primary-type' in release['release-group']:
+        rel_primarytype = release['release-group']['primary-type']
+        if rel_primarytype:
```

Comment:
You check the string not emptiness in that case but not for secondarytype. Intentional ?